tests/accessor/local_accessor_api_fp16.cpp

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides sycl::local_accessor api test for the sycl::half type
+//
+*******************************************************************************/
+#include "../common/disabled_for_test_case.h"
+#include "catch2/catch_test_macros.hpp"
+
+// FIXME: re-enable when sycl::local_accessor is implemented
+#if !defined(__HIPSYCL__) && !defined(__COMPUTECPP__) && \
+    !defined(__SYCL_COMPILER_VERSION)
+#include "accessor_common.h"
+#include "local_accessor_api_common.h"
+#endif
+
+namespace local_accessor_api_fp16 {
+
+DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCpp, DPCPP)
+("sycl::local_accessor api. fp16 type", "[accessor]")({
+  using namespace local_accessor_api_common;
+
+  auto queue = sycl_cts::util::get_cts_object::queue();
+  if (queue.get_device().has(sycl::aspect::fp16)) {
```

Comment:
Would you mind to add `else` branch that will say the reason of test skipping
with the following message, for example: "Device does not support double precision floating point operations"